lib/paginate-request.js

```diff
@@ -10,7 +10,7 @@ module.exports =  async (request, query) => {
     let page = query.page + 1; // since you already made one request above
     const requests = [];
     while (page <= totalPages) {
-      requests.push(request(merge(query), { page }));
+      requests.push(request(merge(query, { page, offset: page*query.perPage })));
```

Comment:
offset is only used for data table rows, I would suggest maybe passing in an option to use offset instead of per page and by default use offset.

Reply:
By default use offset? Will files/experience understand offset if it defaults to that?